x/ccv/provider/types/proposal.go

```diff
@@ -145,17 +163,7 @@ func (cccp *ConsumerAdditionProposal) ValidateBasic() error {
 		return errorsmod.Wrap(ErrInvalidConsumerAdditionProposal, "unbonding period cannot be zero")
 	}
 
-	// Top N corresponds to the top N% of validators that have to validate the consumer chain and can only be 0 (for an
-	// Opt In chain) or in the range [50, 100] (for a Top N chain).
-	if cccp.Top_N != 0 && (cccp.Top_N < 50 || cccp.Top_N > 100) {
-		return errorsmod.Wrap(ErrInvalidConsumerAdditionProposal, "Top N can either be 0 or in the range [50, 100]")
-	}
-
-	if cccp.ValidatorsPowerCap != 0 && cccp.ValidatorSetCap > 100 {
-		return errorsmod.Wrap(ErrInvalidConsumerAdditionProposal, "validators' power cap has to be in the range [1, 100]")
-	}
-
-	return nil
+	return ValidatePSSFeatures(cccp.Top_N, cccp.ValidatorsPowerCap, ErrInvalidConsumerAdditionProposal)
```

Comment:
nits: I'd rather handle the error here and wrap it with `ErrInvalidConsumerAdditionProposal`.